### PR TITLE
Propose more code owners

### PR DESCRIPTION
**Pull request by plexoos**

### CODE REVIEW

**Review by nigmatkulov — COMMENTED** [review]
I also propose to add: tnonaka
as a code owner for: /StRoot/StRefMultCorr

**Comment by plexoos** [comment]
Sure. Is it the actual github user name?

**Comment by plexoos** [comment]
@zlchang  and @veprbl please confirm if you are ok with the proposed change

**Comment by zlchang** [comment]
Hi Dmitry,

I’m good with this change.

Zilong

Sent from my iPhone

On Jun 2, 2021, at 1:20 PM, Dmitri Smirnov ***@***.***> wrote:

﻿

@zlchang<https://github.com/zlchang> and @veprbl<https://github.com/veprbl> please confirm if you are ok with the proposed change

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -14,6 +14,7 @@
 /StRoot/StGenericVertexMaker @plexoos     @perevbnlgov
 /StRoot/StHeavyPool          @starsdong   @ullrich-bnl @fgeurts
 /StRoot/StHighPtTagsMaker    @ullrich-bnl
+/StRoot/StJetMaker           @zlchang     @veprbl
```
Comment (veprbl) [inline .github/CODEOWNERS:18]:
```suggestion
/StRoot/StJetFinder           @zlchang     @veprbl
/StRoot/StJetMaker           @zlchang     @veprbl
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -22,6 +23,7 @@
 /StRoot/StPico*              @nigmatkulov @marrbnl
 /StRoot/StRefMultCorr        @nigmatkulov
 /StRoot/StSpectraTagsMaker   @ullrich-bnl
+/StRoot/StSpinPool           @zlchang     @veprbl
```
Comment (veprbl) [inline .github/CODEOWNERS]:
```suggestion
/StRoot/StSpinPool           @akioogawa @zlchang     @veprbl
```

**Comment by zlchang** [comment]
Hi Dmitry,

Can you also add me to own  /StRoot/StTriggerUtilities

Zilong

On Jun 2, 2021, at 11:07 PM, Dmitri Smirnov ***@***.***> wrote:

﻿

Merged #7<https://github.com/star-bnl/star-sw/pull/7> into main.

**Comment by plexoos** [comment]
Sure. Will do

**Pull request merged**